x-pack/plugins/snapshot_restore/public/application/components/policy_form/steps/step_logistics.tsx

```diff
@@ -420,7 +417,7 @@ export const PolicyStepLogistics: React.FunctionComponent<StepProps> = ({
                 defaultMessage="Use cron expression. {docLink}"
                 values={{
                   docLink: (
-                    <EuiLink href={documentationLinksService.getCronUrl()} target="_blank">
+                    <EuiLink href={docLinks.links.watcher.cronSchedule} target="_blank">
```

Comment:
For SLM, I think it might make more sense to point to these docs https://www.elastic.co/guide/en/elasticsearch/reference/current/cron-expressions.html rather than watcher.

Reply:
Thanks, I've changed that URL